heal.js

```diff
@@ -25,11 +25,21 @@ var ClusterManager = require('./lib/cluster.js');
 var assertNoError = require('./lib/util.js').assertNoError;
 var program = require('commander');
 
+function safeParseInt(number, defaultValue) {
+    number = parseInt(number, 10);
+    if (isNaN(number)) {
+        return defaultValue;
+    } else {
+        return number;
+    }
+}
+
 function main() {
     program
         .description('Start a partition heal coordinated by the target node')
         .option('--tchannel-v1')
-        .option('--verbose')
+        .option('--quiet')
+        .option('--tries <tries>', 'Number of times to try the heal', safeParseInt, 10)
```

Comment:
If the user doesn't specify `--tries`, should we have a sensible default?

Also when should someone tweak the retry limit? Is that in order to heal multiple partitions?